packages/components/psammead-episode-list/src/fixtures.jsx

```diff
@@ -43,75 +55,201 @@ export const exampleEpisodes = [
-              {episode.episodeTitle || `${episode.date}, ${episode.time}`}
-            </EpisodeList.Description>
-            <VisuallyHiddenText>, </VisuallyHiddenText>
-            <VisuallyHiddenText>
-              {` ${episode.durationLabel} ${formatDuration({
-                duration: episode.duration,
-                format: episode.duration.includes('H') ? 'h,mm,ss' : 'mm,ss',
-                locale: episode.locale,
-              })} `}
-            </VisuallyHiddenText>
-            <EpisodeList.Metadata>
-              <span aria-hidden="true">
-                {` ${episode.durationLabel} ${formatDuration({
+export const exampleVideoEpisodes = exampleEpisodes.map(episode => ({
+  ...episode,
+  image: 'https://ichef.bbci.co.uk/images/ic/768x432/p08b22y1.png',
+  altText: 'BBC News Afrique',
+}));
+
+export const exampleRtlVideoEpisodes = rtlEpisodes.map(episode => ({
+  ...episode,
+  image: 'https://ichef.bbci.co.uk/images/ic/768x432/p08b22y1.png',
+  altText: 'BBC News Afrique',
+}));
+
+const StyledSpan = styled.span`
+  padding-left: 8px;
+  padding-right: 8px;
+`;
+
+const Spacer = styled.div`
+  background: ${({ darkMode }) => (darkMode ? C_MIDNIGHT_BLACK : 'unset')};
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  padding: 0 ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_SPACING_DBL};
+  }
+`;
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-bottom: 0;
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_DBL};
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  ${({ darkMode }) => darkMode && `color: ${C_WHITE}`}
+`;
+
+const SurroundingComponents = ({
+  children,
+  script,
+  service,
+  dir,
+  darkMode,
+}) => (
+  <Spacer darkMode={darkMode}>
+    <StyledSectionLabel
+      script={script}
+      service={service}
+      dir={dir}
+      darkMode={darkMode}
+      {...(darkMode ? { backgroundColor: C_MIDNIGHT_BLACK } : {})}
+    >
+      Recent Episodes
+    </StyledSectionLabel>
+    {children}
+  </Spacer>
+);
+
+export const renderEpisodes = ({
+  episodes,
+  script,
+  service,
+  dir,
+  withSurroundingComponents,
+  darkMode,
+}) => {
+  const Wrapper = withSurroundingComponents
+    ? SurroundingComponents
+    : React.Fragment;
+  return (
+    <Wrapper
+      {...(withSurroundingComponents ? { script, service, dir, darkMode } : {})}
+    >
+      <EpisodeList script={script} service={service} dir={dir}>
+        {episodes.map(episode => (
+          <EpisodeList.Link key={episode.id} href={episode.url}>
+            <EpisodeList.Episode>
+              {/* eslint-disable-next-line jsx-a11y/aria-role */}
+              <span role="text">
+                <VisuallyHiddenText>Audio, </VisuallyHiddenText>
+                <EpisodeList.Title className="underlined_hover fade_visited">
```

Comment:
These magic class names could be documented or replaced with props

Reply:
if we use class names should we maybe use BEM naming convention? e.g. `episode-list__title--hover` and `episode-list__title--visited`